README.md

```diff
@@ -57,28 +59,106 @@ systemctl restart jans-auth
+```
+- Open TUI: `python3 /opt/jans/jans-cli/jans_cli_tui.py`
+- Navigate to the `Agama` tab and then select `"Upload project"`. Choose the `.gama` file
 - Wait for about one minute and then select the row in the table corresponding to this project
-- Press `d` and ensure there were not deployment errors
+- Press `v` and ensure there were not deployment errors
 - Pres `ESC` to close the dialog
 
 ![TUI_AGAMA_DEPLOY](https://github.com/GluuFederation/agama-passkey/assets/86965029/1d6b8cab-ddad-451c-b620-d19be1b7f9e3)
 
 ### Configure Jans Scim
 
-- Once we have deployed the `agama project`, we need to configure the `jans scim` parameters, then we proceed to create a new user `jans scim` with scope `https://jans.io/scim/fido2.read` and `https://jans.io/scim/fido2.write`.
+- Once we have deployed the `agama project`, we need to configure the `jans scim` parameters, then we proceed to create
+  a new client `jans scim` with scope `https://jans.io/scim/fido2.read` and `https://jans.io/scim/fido2.write`.
+
+You can create the client using the registration web service.
+
+**Request**
+
+```
+curl --location 'https://YOUR_DOMAIN/jans-auth/restv1/register' \
+--header 'Content-Type: application/json' \
+--data '{
+  "client_name": "Scim Agama Client",
+  "scope": [
+    "https://jans.io/scim/fido2.read",
+    "https://jans.io/scim/fido2.write"
+  ],
+  "grant_types": [
+    "client_credentials"
+  ],
+  "token_endpoint_auth_method": "client_secret_basic",
+  "response_types": [
+    "code"
+  ]
+}'
+```
+
+**Response**
+
+```
+{
+  "allow_spontaneous_scopes": false,
+  "application_type": "web",
+  "rpt_as_jwt": false,
+  "registration_client_uri": "https://jenkins-dev1.jans.io/jans-auth/restv1/register?client_id=27975f1c-eee6-4bf8-b393-5fb47d44c566",
+  "tls_client_auth_subject_dn": "",
+  "run_introspection_script_before_jwt_creation": false,
+  "registration_access_token": "ca1e8640-d59a-4603-a42c-936e69385101",
+  "client_id": "<YOUR_CLIENT_ID>",
+  "client_secret": "<YOUR_SECRET_KEY>",
+  "token_endpoint_auth_method": "client_secret_basic",
+  "scope": "https://jans.io/scim/fido2.read https://jans.io/scim/fido2.write",
+  "client_id_issued_at": 1710469308,
+  "backchannel_logout_session_required": false,
+  "client_name": "Scim custom client",
+  "par_lifetime": 600,
+  "spontaneous_scopes": [],
+  "id_token_signed_response_alg": "RS256",
+  "access_token_as_jwt": false,
+  "grant_types": [
+    "authorization_code",
+    "refresh_token",
+    "client_credentials"
+  ],
+  "subject_type": "pairwise",
+  "authorization_details_types": [],
+  "additional_token_endpoint_auth_methods": [],
+  "keep_client_authorization_after_expiration": false,
+  "require_par": false,
+  "redirect_uris_regex": "",
+  "additional_audience": [],
+  "frontchannel_logout_session_required": false,
+  "client_secret_expires_at": 1710555708,
+  "access_token_signing_alg": "RS256",
+  "response_types": [
+    "code"
+  ]
+}
+```
+
+or you can use the `TUI`
```

Comment:
reference jans docs... don't repeat jans docs ...

Reply:
Remove this section, the first example of client creation should suffice.